doc/api/process.md

```diff
@@ -3464,6 +3464,19 @@ throw an error.
 Using this function is mutually exclusive with using the deprecated
 [`domain`][] built-in module.
 
+## `process.sourceMapsEnabled`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
```

Comment:
Is it really experimental?

Reply:
Given that source maps are still considered experimental this needs to be also. That said, it may be about time to graduate them.

Reply:
That makes sense.

Reply:
Yes, I made this as experimental because of that reason.